### PR TITLE
Fix Avail call and UI

### PR DESCRIPTION
**Pull request by gguijarro-c-chwy**
missing await was the culprit. 

Some cleanup and UI updates

### CODE REVIEW

File: lib/ui/settings/settings_page.dart [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import 'package:flutter/material.dart';
```
Comment (gguijarro-c-chwy) [inline lib/ui/settings/settings_page.dart:1]: i messed up and pushed UI code i was doing for settings on this PR...

Comment (gguijarro-c-chwy) [reply lib/ui/settings/settings_page.dart:1]: image

**Comment by n13** [comment]
Great fix thanks

**Review by n13 — APPROVED** [review]
LGTM

**Pull request merged**